### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1019] [Galaxy Trivia] [answered]
**Siberia Bow Mill**
**In zee inn id daze**
**Rain Van**
**Hairy Camera**

**Comment by peterbaccaro** [comment]
1. Robbie Williams
2. Zinedine Zidane
3. Nirvana
4. Mariah Carey

The odd one out is **Zinedine Zidane** as he is a footballer and the others are singers.

**Comment by thuongtruong1009** [comment, marked as answer]
The odd one out is "Hairy Camera" because it does not follow the pattern of the other options. The other options consist of two words that are related or can be paired together (e.g., Siberia Bow, Mill in zee, Rain Van), while "Hairy Camera" does not form a coherent or related pair.
